linting.mk

```diff
@@ -0,0 +1,27 @@
+### linting options below
```

Comment:
Would it not be better to use bingo for this as it skips the need to have /tmp/90poe/ folder I guess?